sdcm/cluster.py

```diff
@@ -3662,11 +3672,22 @@ def update_scylla_packages(node, _queue):
             node.remoter.run('tar -xvf /tmp/scylla/*.tar.gz -C /tmp/scylla/', ignore_status=True, verbose=True)
 
             # replace the packages
-            logging.info("Installing rpms")
-            node.remoter.run('yum list installed | grep scylla')
-            node.remoter.sudo('rpm -URvh --replacepkgs --replacefiles /tmp/scylla/*.rpm',
-                              ignore_status=False, verbose=True)
-            node.remoter.run('yum list installed | grep scylla')
+            self.log.debug(f'Node distro is {node.distro}, will check the package suffix received')
+            if node.distro.is_rhel_like:
+                check_package_suites_distro(node, 'rpm')
+                self.log.info('Installed RPMs before replacing with new RPM files')
+                node.remoter.run('yum list installed | grep scylla', verbose=True)
+                self.log.debug('Will replace the RPM files with the ones provided as a parameter to the test')
+                node.remoter.sudo('rpm -URvh --replacefiles /tmp/scylla/*.rpm', ignore_status=False, verbose=True)
+                self.log.info('Installed RPMs after replacing with new RPM files')
+                node.remoter.run('yum list installed | grep scylla', verbose=True)
+            elif node.distro.is_ubuntu:
```

Comment:
how about debian?

`elif node.distro.is_ubuntu or node.distro.is_debian:`

Reply:
well, this is not something that we test in a frequent manner, nor we have AMIs built with Debian, but if once decided we want to test an AMI with Debian installed, it will be super easy to add it, but for now, i think it is not necessary

Reply:
> well, this is not something that we test in a frequent manner, nor we have AMIs built with Debian,

You are right.

> but if once decided we want to test an AMI with Debian installed, it will be super easy to add it, but for now, i think it is not necessary